### DISCUSSION TITLE
Captcha: You have been blocked

### DISCUSSION THREAD

**Discussion by DerVomHof** [ahivert/tgtg-python#240] [Q&A]
After a few queries to the api got the 403 error message with this link:
(https://geo.captcha-delivery.com/captcha/?initialCid=AHrlqAAAAAMA_DWjbHD3LREAX1rBqA==&cid=6WOOKOg1iW_gX6FV666uUrhWMNmFTkHemJLFtgW7ptNOGF2LPfjqnfwDTKFRxGqmMKT0mJJGhmM6byvDXCmFtbJUUl2cz67WVdFQy0_RL6wMVCJcUbLxcRSkNm4be_jS&referer=https%3A%2F%2Fapptoogoodtogo.com%2Fapi%2Fauth%2Fv3%2Ftoken%2Frefresh&hash=1D42C2CA6131C526E09F294FE96F94&t=fe&s=35558&e=63da734ae6c1ed24fa6a4f1531fbde67ef471bf2aca26aa8120ab27864b254a7).
Apparently my requests were blocked and on the website you can solve a captcha, but this only ensures that the page displays a loading bar.  Now to my question, is there a new maximum number of queries you can make, or is there another way around this?

**Comment by angpaz** [comment]
Same issue here, after waiting some time it works again without being blocked just to getting blocked again.... annoying if you want to poll quite frequently ( I am trying one request per 1 min). Any chance to avoid the captcha by rotating ip's or simple changes of the request header?

**Reply by matrop** [reply]
I had a similar issue and described a workaround in #241. Does this help you?

**Comment by frarzo** [comment]
Is there a known query rate? So far I've been able to keep a request every 5 mins without a problem for a couple of days, but I'd like to bump it up a little, like every 3 mins but I'd also like to not get blocked :)
